integration_tests/test_upgrade.py

```diff
@@ -163,4 +173,16 @@ def test_cosmovisor_upgrade(custom_cronos: Cronos):
     assert old_base_fee == w3.eth.get_block(old_height).baseFeePerGas
 
     # check eth_call works on older blocks
-    contract.caller(block_identifier=target_height - 2).balanceOf(ADDRS["validator"])
+    assert old_erc20_balance == contract.caller(block_identifier=old_height).balanceOf(
+        ADDRS["validator"]
+    )
+
+    assert not cli.evm_params()["extra_eips"]
```

Comment:
```suggestion
    assert not cli.evm_params().extra_eips
```

Reply:
changed, this needs to be `cli.evm_params()['params']['extra_eips']`